sdk/cosmos/azure-cosmos-spark_3_2-12/dev/README.md

```diff
@@ -43,8 +43,8 @@ The release process is currently manual.
 Download the spark artifacts or build locally:
 ```bash
 mvn -e -DskipTests -Dgpg.skip -Dmaven.javadoc.skip=true -Dcodesnippet.skip=true -Dspotbugs.skip=true -Dcheckstyle.skip=true -Drevapi.skip=true -pl ,azure-cosmos -am clean install
-mvn -e -DsparkShading -DskipTests -Dgpg.skip -Dmaven.javadoc.skip=true -Dcodesnippet.skip=true -Dspotbugs.skip=true -Dcheckstyle.skip=true -Drevapi.skip=true -pl ,azure-cosmos-spark_3-1_2-12 clean install
-mvn -e -DsparkShading -DskipTests -Dgpg.skip -Dmaven.javadoc.skip=true -Dcodesnippet.skip=true -Dspotbugs.skip=true -Dcheckstyle.skip=true -Drevapi.skip=true -pl ,azure-cosmos-spark_3-2_2-12 clean install
+mvn -e -DskipTests -Dgpg.skip -Dmaven.javadoc.skip=true -Dcodesnippet.skip=true -Dspotbugs.skip=true -Dcheckstyle.skip=true -Drevapi.skip=true -pl ,azure-cosmos-spark_3-1_2-12 clean install
```

Comment:
We are removing spark shading here as well, which I don't think is correct. 
Spark packages are shaded intentionally, and that's how customers use them.

Reply:
From the other comment shading Spark is always happening now